### DISCUSSION TITLE
Nuxt Layout change value depending on child

### DISCUSSION THREAD

**Discussion by jTiKey** [nuxt/nuxt#26274] [Questions]
layouts/default.vue

```
<div>
    {{ child_value }}
<div>
<slot/>
```

page/child.vue

```
<script setup>
const child_value  = 'some value'
<script>
```

How do I change the child_value from child.vue?

**Comment by mfazail** [comment]
Probably use [vuejs provide/inject](https://vuejs.org/guide/components/provide-inject.html) or use [pinia](https://pinia.vuejs.org/)

**Comment by jTiKey** [comment]
> Probably use [vuejs provide/inject](https://vuejs.org/guide/components/provide-inject.html) or use [pinia](https://pinia.vuejs.org/)

pinia won't work because I would need to change the state on all pages on each mount. I'll try provide inject, thanks@
